### PR TITLE
fix: rebuild artifacts

### PR DESCRIPTION
**Pull request by foxpy**

### CODE REVIEW

File: artifacts/checksums.txt [hunk 1]
```diff
@@ -1,10 +1,13 @@
-c5a2673b74cda51bd2628804856117291ed555867daf66f543e084dccc694576  cw4_voting.wasm
+e1f198ec554470bd05f4081cf2309035a1369152b147bdc2ca2fad7120364641  cw4_group.wasm
```
Comment (sotnikov-s) [inline artifacts/checksums.txt:1]: does it belong here? I'd move it to the integration tests repo instead

Comment (foxpy) [reply artifacts/checksums.txt:1]: I have no idea, the only thing I have done here is run `make build`

Comment (sotnikov-s) [reply artifacts/checksums.txt:1]: sorry, I meant `cw4_group.wasm`. it shouldn't be a result of `make build`. it's not even in the `checksum.txt` file

Comment (oopcode) [reply artifacts/checksums.txt:1]: because it's required for the test script

**Pull request merged**